src/DSharpPlus.Examples.Commands.Basics/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using DSharpPlus.Commands;
+using DSharpPlus.Commands.Processors.SlashCommands;
+using DSharpPlus.Commands.Processors.TextCommands;
+using DSharpPlus.Commands.Processors.TextCommands.Parsing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace DSharpPlus.Examples.Commands.Basics
+{
+    public static class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            ConfigurationBuilder configurationBuilder = new();
+            // you can also use environment vars, just uncomment this
+            // configurationBuilder.AddEnvironmentVariables("");
```

Comment:
// right click on config.json, click properties, and in Copy to Output Directory select Copy if newer